### PR TITLE
Telemetry for CLUE on Python Side

### PR DESCRIPTION
**Pull request by vandyliu**
# Description:
- Added telemetry for 
  - all adafruit_clue functions
  - adafruit slideshow (when initialized)
  - tile_grid (when shapes are drawn on screen) 

## Type of change
- [ ] New feature (non-breaking change which adds functionality)

# Limitations:

Please describe limitations of this PR

# Testing:

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] My code has been formatted with `npm run format` and passes the checks in `npm run check`
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/clue/adafruit_clue.py [hunk 1]
```diff
@@ -279,6 +284,7 @@ def acceleration(self):
           while True:
             print("Accel: {:.2f} {:.2f} {:.2f}".format(*clue.acceleration))
         """
+        telemetry_py.send_telemetry(TelemetryEvent.CLUE_API_ACCELERATION)
```
Comment (nasadigital) [inline src/clue/adafruit_clue.py:287]: It seems that `CLUE_API_ACCELERATION` was not defined in the enum.

**Review by nasadigital — APPROVED** [review]
LGTM

File: src/clue/adafruit_clue.py (hunk 1 above)
Comment (vandyliu) [reply src/clue/adafruit_clue.py:287]: fixed

**Review by andreamah — APPROVED** [review]
looks great 💖

**Review by xnkevinnguyen — APPROVED** [review]
LGTM, Thanks for adding telemetry to unimplemented functions. This will help next interns prioritize features

**Pull request merged**